histomicsui/web_client/templates/panels/metadataWidget.pug

```diff
@@ -0,0 +1,30 @@
+.s-panel-title-container
+  span.s-panel-controls
+    if collapsed
+      i.icon-down-open
+    else
+      i.icon-up-open
+
+if (accessLevel >= AccessType.WRITE)
+  .btn-group.pull-right
+    button.g-widget-metadata-add-button.btn.btn-sm.btn-primary.btn-default.dropdown-toggle(data-toggle="dropdown", title="Add Metadata")
+      i.icon-plus
+    ul.dropdown-menu.pull-right(role="menu")
+      li(role="presentation")
+        a.g-add-simple-metadata(role="menuitem")
+          | Simple
+      li(role="presentation")
+        a.g-add-json-metadata
+          | JSON
+.s-panel-title-container
```

Comment:
Having two `.s-panel-title-container` divs seems surprising.  Further, the title bar of the panel appears taller than other panels.  I expect that this was done partly because clicking on the title container causes the panel to open and close, and this prevented the Add button from working.

Since the panel title toggle is behavior inherited from slicer_cli_web, we can modify it: see https://github.com/girder/slicer_cli_web/pull/110.

I think it would be better if this entire pug file were something more like (some comments in line):
```
.s-panel-title-container
  span.s-panel-title.h-panel-title
    i.icon-tags
    block title                            # we actually don't need the block here, since that allows other panel to inherit from this one
      |  #{title}                            # note two spaces before #{title} for consist look with other panels
  if (firstKey && firstValue)
    span.g-widget-metadata-header-key-value #{firstKey+' '+firstValue+'  '}

  span.s-panel-controls
    if (accessLevel >= AccessType.WRITE)
      span.s-no-panel-toggle                    # new class introduced in the mentioned PR
        button.g-widget-metadata-add-button.btn.btn-sm.btn-primary.btn-default.dropdown-toggle(data-toggle="dropdown", title="Add Metadata")
          i.icon-plus
        ul.dropdown-menu.pull-right(role="menu")
          li(role="presentation")
            a.g-add-simple-metadata(role="menuitem")
              | Simple
          li(role="presentation")
            a.g-add-json-metadata
              | JSON

    if collapsed
      i.icon-down-open
    else
      i.icon-up-open
```

This would also require some css changes: 
```
  .s-panel-title-container
    .s-no-panel-toggle
      position relative
```
plus removing the `.s-panel-controls` `margin-top` and setting `button.g-widget-metadata-add-button` `float none`.

I haven't yet checked that such changes don't mess up other styles in the panel.

If you are using the docker containers, you can try out the PR from slicer_cli_web by
```
docker exec -it histomicstk_histomicstk bash
cd /opt/slicer_cli_web
git pull
git checkout origin/add-no-toggle-class
```
and rebuilding the girder web client as usual.

Reply:
Thank you David, I will work on it and try to fix things.